src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -134,6 +134,28 @@ export const LongButtonText: StoryObj<Args> = {
   },
 };
 
+export const Disabled: StoryObj<Args> = {
+  render: () => (
+    <div
+      className={clsx(
+        styles['trigger--spacing-top'],
+        styles['trigger--spacing-left'],
+      )}
```

Comment:
One noteworthy thing here is any spacing/placement styling has to be placed on a wrapper around the `Tooltip`, not the `children`, because of the new wrapper. That could be pretty confusing for people.

Reply:
I added some info about this to the prop comment.